schema/deploy/policies/ciip_user_policies_003_restrict_permissions.sql

```diff
@@ -0,0 +1,45 @@
+-- Deploy ggircs-portal:policies/ciip_user_policies_003_restrict_permissions to pg
+-- requires: policies/ciip_user_policies_002_recreate_after_type_change
```

Comment:
Should I always be requiring the previous version of the function in the new one?

Reply:
For these numbered migrations I always require the previous one, it may not _actually_ depend on the previous change, but it does have a dependency somewhere along the chain and if we make all subsequent numbered migrations depend on the previous numbered migration then we will always have a transitive dependency to wherever it does have a dependency in the chain.